spyder_kernels/console/kernel.py

```diff
@@ -252,10 +252,11 @@ def load_data(self, filename, ext):
         if error_message:
             return error_message
 
-        for key in list(data.keys()):
-            new_key = fix_reference_name(key, blacklist=list(glbs.keys()))
-            if new_key != key:
-                data[new_key] = data.pop(key)
+        if not override:
+            for key in list(data.keys()):
```

Comment:
Why do  we convert to list?

`for key in data.keys():`this should work just fine, right?

Reply:
@dalthviz ?

Reply:
so this does not work

`for key in data.keys():`  ?

Reply:
I was just checking if this change somehow affected the tests in AppVeyor but is unrelated